src/data/setoid/partition.lean

```diff
@@ -67,8 +67,9 @@ lemma classes_ker_subset_fiber_set {β : Type*} (f : α → β) :
   (setoid.ker f).classes ⊆ set.range (λ y, {x | f x = y}) :=
 by { rintro s ⟨x, rfl⟩, rw set.mem_range, exact ⟨f x, rfl⟩ }
 
-lemma finite_classes_ker {α β : Type*} [finite β] (f : α → β) : finite (setoid.ker f).classes :=
-by { classical, exact finite.set.subset _ (classes_ker_subset_fiber_set f) }
+lemma finite_classes_ker {α β : Type*} [finite β] (f : α → β) :
+  (setoid.ker f).classes.finite :=
+(set.finite_range _).subset $ classes_ker_subset_fiber_set f
```

Comment:
Given `card_classes_ker_le` it might make sense for there to be a `finite` instance for `(setoid.ker f).classes`, but this seems like it could be deferred to whenever someone else might need it.

bors r+